.github/workflows/autobuild.yml

```diff
@@ -149,13 +149,13 @@ jobs:
 
           - config_name:            iOS (artifacts)
             target_os:              ios
-            building_on_os:         macos-10.15
-            base_command:           QT_VERSION=5.15.2 ./.github/autobuild/ios.sh
+            building_on_os:         macos-11
+            base_command:           QT_VERSION=5.15.2 SIGN_IF_POSSIBLE=1 ./.github/autobuild/ios.sh
```

Comment:
Could you use the presence of a certificate in the environment to determine whether to sign, so not needing another parameter?

Reply:
I would rather follow the lead as being done for macOS, which uses that flag. Or we can remove from both macOS and iOS but to have in just one feels weird.